neqo-transport/src/connection/params.rs

```diff
@@ -223,12 +226,23 @@ impl ConnectionParameters {
         self
     }
 
-    pub fn get_queued_datagrams(&self) -> usize {
-        self.queued_datagrams
+    pub fn get_outgoing_datagrams_queue(&self) -> usize {
```

Comment:
You have a datagram queue or a queue of datagrams:

```suggestion
    pub fn get_outgoing_datagram_queue(&self) -> usize {
```